js/services/engine.ts

```diff
@@ -980,7 +981,8 @@ export const move = (outcome: any, flowKey: string)  => {
 
     const invokeRequest = Json.generateInvokeRequest(
         State.getState(flowKey),
-        Model.getInvokeType(flowKey),
+        // 'SYNC' requests would not be able to move down an outcome
+        Utils.isEqual(Model.getInvokeType(flowKey), 'SYNC', true) ? 'FORWARD' : Model.getInvokeType(flowKey),
```

Comment:
Why should it go 'FORWARD' if it is set to 'SYNC'? Shouldn't it make a sync request (not move)?

Reply:
Move is called when the user has clicked a button and wants to move somewhere.
'SYNC' is coming back from the engine last because a page condition just updated the page, so it was a 'SYNC' request last, but that doesn't meant the engine wants the UI to do a SYNC next, especially if the U wants to move somewhere.